service-env-defaults

```diff
@@ -0,0 +1,224 @@
+###########################################################################
```

Comment:
I think splitting out these from `env-defaults` makes it harder to deal with environment variables and harder for people to add new harmony services. I think my preference would be to just have a single `env-defaults` file at the top level - even for all the tasks and submodules we end up creating. You'd still define your expected environment for each task and module in an `env.ts` file but we wouldn't splinter the defaults all throughout the repo - just one place.

You still have the benefit of seeing which environment variables are used by which with the separate `env.ts` files, and we can still have separate modules to minimize the dependencies required for each new task/service we create.

Reply:
Reverted back to the main env-defaults